src/elements/content-explorer/ContentExplorer.tsx

```diff
@@ -410,13 +424,25 @@ class ContentExplorer extends Component<ContentExplorerProps, State> {
             // Set limit to the query for pagination support
             metadataQueryClone.limit = DEFAULT_PAGE_SIZE;
         }
+
+        metadataQueryClone.order_by = [
+            {
+                field_key: sortBy,
+                direction: sortDirection,
+            },
+        ];
         // Reset search state, the view and show busy indicator
         this.setState({
             searchQuery: '',
             currentCollection: this.currentUnloadedCollection(),
             view: VIEW_METADATA,
         });
-        this.metadataQueryAPIHelper = new MetadataQueryAPIHelper(this.api);
+
+        this.metadataQueryAPIHelper = new MetadataQueryAPIHelper(
+            this.api,
+            isFeatureEnabled(features, 'contentExplorer.metadataViewV2'),
+        );
```

Comment:
i was thinking you can call the `features/metadata-based-view/MetadataQueryAPIHelper` for v1 and the new file for v2. that way you wouldn't have to write specific logic in `MetadataQueryAPIHelper` to account for both scenarios

Reply:
ah got it